specs/core/0_beacon-chain.md

```diff
@@ -855,76 +850,99 @@ def split(values: List[Any], split_count: int) -> List[Any]:
     ]
 ```
 
+#### `get_committees_per_slot`
+
+```python
+def get_committees_per_slot(active_validator_count: int) -> int:
+    return max(
+        1,
+        min(
+            SHARD_COUNT // EPOCH_LENGTH,
+            len(active_validator_indices) // EPOCH_LENGTH // TARGET_COMMITTEE_SIZE,
+        )
+    )
+```
+
 #### `get_shuffling`
 
 ```python
 def get_shuffling(seed: Hash32,
                   validators: List[ValidatorRecord],
-                  crosslinking_start_shard: int,
-                  slot: int) -> List[List[ShardCommittee]]:
+                  slot: int) -> List[List[int]]
     """
-    Shuffles ``validators`` into shard committees seeded by ``randao_mix`` and ``slot``.
+    Shuffles ``validators`` into shard committees seeded by ``seed`` and ``slot``.
+    Returns a list of ``EPOCH_LENGTH * committees_per_slot`` committees where each
+    committee is itself a list of validator indices.
     """
 
     # Normalizes slot to start of epoch boundary
     slot -= slot % EPOCH_LENGTH
 
     active_validator_indices = get_active_validator_indices(validators, slot)
 
-    committees_per_slot = max(
-        1,
-        min(
-            SHARD_COUNT // EPOCH_LENGTH,
-            len(active_validator_indices) // EPOCH_LENGTH // TARGET_COMMITTEE_SIZE,
-        )
-    )
+    committees_per_slot = get_committees_per_slot(len(active_validator_indices))
 
     # Shuffle
     seed = xor(randao_mix, bytes32(slot))
```

Comment:
`randao_mix` is undefined, shouldn't it just be `seed = xor(seed, bytes32(slot))`?